### DISCUSSION TITLE
关于搜索引擎

### DISCUSSION THREAD

**Discussion by CiangCing14** [banned-historical-archives/banned-historical-archives.github.io#4032] [General]
建议使用Bing搜索上传sitemap.xml，可以保证全部内容都很快被收录。

**Comment by CiangCing14** [comment]
全部被收录了（案例）：

https://www.bing.com/search?q=site%3Accrd2024.github.io+%E6%AF%9B%E6%B3%BD%E4%B8%9C%E4%B8%BB%E4%B9%89

https://www.bing.com/search?q=site%3Accrd2024.github.io+%E4%B8%8A%E6%B5%B7%E4%BA%BA%E6%B0%91%E5%85%AC%E7%A4%BE+%E5%BC%A0%E6%98%A5%E6%A1%A5

**Reply by banned-historical-archives** [reply]
好像不行，我这里显示空白

**Reply by hrwxl** [reply]
请问解放日报如何批量下载？都是图片形式

**Comment by CiangCing14** [comment]
和谐历史档案馆已经收录了，但是sitemap.xml似乎更新不及时，没有“毛泽东主义”的搜索内容。
